sdk/resourcemanager/Azure.ResourceManager.Core/tests/Unit/SubscriptionProviderIdentifierTests.cs

```diff
@@ -0,0 +1,30 @@
+using NUnit.Framework;
+using System;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    [Parallelizable]
+    public class SubscriptionProviderIdentifierTests
+    {
+        [TestCase("/subscriptions/db1ab6f0-4769-4b27-930e-01e2ef9c123c/providers/Microsoft.Insights")]
```

Comment:
add parent verification
add error scenarios

Reply:
Also add property validations over the SubscriptionProviderIdentifier